docs/agent/changelog.md

```diff
@@ -6,6 +6,14 @@ title: Changelog
 
 ## v3
 
+### ngrok Agent 3.2.0 - \[2024-08-29\]
+
+- Added `--url` agent CLI flag for simpler creation of HTTP(S), TLS, and TCP protocol endpoints.
+- Added `--metadata` and `--description` agent CLI flags when using `--url`.
+- Deprecated `--domain`, `--scheme`, and `--remote-addr` agent CLI flags.
```

Comment:
Could you add a second sentence letting users know what they should use instead of these options?